Assets/Scripts/MapTools/Door.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using UnityEditor.PackageManager;
 using UnityEngine;
 using UnityEngine.SceneManagement;
+using Logger = BoBLogger.Logger;
```

Comment:
at other places using static BobLogger.logger is used 
pls only use one variant

Reply:
see spawnobject as example

Reply:
fixt